pkg/chunk/aws/dynamodb_storage_client.go

```diff
@@ -135,6 +136,9 @@ func (cfg *StorageConfig) RegisterFlags(f *flag.FlagSet) {
 
 	f.Var(&cfg.S3, "s3.url", "S3 endpoint URL with escaped Key and Secret encoded. "+
 		"If only region is specified as a host, proper endpoint will be deduced. Use inmemory:///<bucket-name> to use a mock in-memory implementation.")
+	f.BoolVar(&cfg.S3ForcePathStyle, "s3.force-path-style", false, "Set this to `true` to force the request to use path-style addressing "+
```

Comment:
this explanation is a bit long for here - maybe put it in `docs/arguments.md` ?

Reply:
maybe a bit long here. But I don't think `docs/arguments.md` is a good place though. Some doc to describe the config.yaml maybe a good place, but I don't find one in Cortex.

Reply:
updated. plz help review again.